esmvaltool/cmorizers/obs/cmorize_obs_clara_avhrr.ncl

```diff
@@ -0,0 +1,235 @@
+; #############################################################################
+; ESMValTool CMORizer for CM SAF CLARA-AHRR v2 data
+; #############################################################################
+;
+; Tier
+;    Tier 3: restricted dataset.
+;
+; Source
+;    https://wui.cmsaf.eu/
+;
+; Last access
+;    2021-03-22
+;
+; Download and processing instructions
+;    1) Create ("register") an user account at
+;       https://wui.cmsaf.eu/safira/action/viewLogin?menuName=NUTZER_HOME
+;    2) login (same URL as above)
+;    3) Search data using search form at
+;    https://wui.cmsaf.eu/safira/action/viewProduktHome?menuName=PRODUKT_HOME
+;
+;      - Product group: Climate Data Records
+;      - Product family: CLARA-A ed. 2.1
+;      - Product name: CFC - Factional cloud cover
+;                      IWP - Ice water path
+;                      LWP - Liquid water path
+;      - Area: Global
+;      - Temporal resolution: Monthly
+;
+;    4) Select "CLARA-A ed. 2.1 AVHRR on polar orbiting satellites" from
+;       list of results.
+;    5) Click on "Add to order cart"
+;    6) Follow download instructions in automatic email received when data
+;       are ready for download.
+;    7) Untar all .tar files into a single directory.
+;
+; Modification history
+;    20210323-laue_axel: written.
+;
+; #############################################################################
+loadscript(getenv("esmvaltool_root") + \
+           "/esmvaltool/cmorizers/obs/interface.ncl")
+
+begin
+
+  ; Script name (for logger)
+  DIAG_SCRIPT = "cmorize_obs_clara_avhrr.ncl"
+
+  ; Source name
+  OBSNAME = "CLARA-AVHRR"
+
+  ; Tier
+  TIER = 3
+
+  ; Period
+  YEAR1 = 1982
+  YEAR2 = 2018
+
+  ; Selected variable (standard name)
+  VAR = (/"clt", "clivi", "clwvi"/)
```

Comment:
Same: does this only work if you have all the variables? What if I just want `clt`?

Reply:
Similar to the time range, this could also be changed if a user (probably rather a developer) would only want one or two of these variables. But if people want to run the new cloud recipe (I would guess this will be the main application for running this CMORizer), they will need all three variables.